_utils/vault.py

```diff
@@ -481,6 +481,15 @@ def ha_status(self):
         """
         return self._get('/v1/sys/leader').json()
 
+    def get_lease(self, lease_id):
+        try:
+            lease = self.write('sys/leases/lookup', lease_id=lease_id)
+        except InvalidRequest:
+            log.exception('The specified lease is not valid')
+            lease = None
```

Comment:
Why are we setting this to `None`?

Reply:
Because if the exception gets raised it means that there isn't any lease

Reply:
Because if the exception gets raised it means that there isn't any lease